### ISSUE TITLE
```swipeViewWithAccessibilityLabel:inDirection:``` not working on iOS 12

### ISSUE THREAD

**Issue by billburgess** [kif-framework/KIF#1093]
This may be somewhat related to #1090 but was asked to open a new issue. For whatever reason this function does not always work on CircleCI (Xcode 10/12.0 and 10.1/12.1 tested). I cannot recreate the issue locally which is why this is so maddening. I have a UISwitch set in a ```UITableViewCell -accessoryView```. The switch is given an accessibilityLabel which is found every time using ```-waitForViewWithAccessibilityLabel```. But the swipe in direction fails. My tests not even erring on this, they are assuming it works, but the switch never activates and the step following fails after timeout looking for the items that should show after the switch activates.

I will do my best and can submit changes to run on CI as needed. I can't reproduce locally unfortunately. Let me know what I can do to help.

**Comment by mikelupo** [comment]
@billburgess is it possible that the swipe is happening to the cell instead of the switch?  And for some awful reason it works locally (perhaps erroneously)? What if you try accessibilityIdentifier instead? Sorry if it seems I’m grasping at straws.

**Comment by billburgess** [comment]
Crazy thing is, I have a very similar setup in a couple of my views with a main switch toggling on/off to reveal more information. All other views seem to work with same setup. Just this one view I have is behaving differently. Gatekeeper to a lot of other views that leads to more failures. Will run with a manual ```setSwitch: element: On:``` keying on the ```-accessibilityIdentifier``` and see how that goes.

**Comment by mikelupo** [comment]
@billburgess lots of people use KIF to toggle switches. It’s something that you’re doing with the table or cell. Just for grins, what if you take the switch out of the table? 

Other musings...
Are you setting the label/identifier in code or in the storyboard? If In the code, and it’s rendering cells that are scrolling into view, it could be that it’s not fully rendered and properties are not applied yet..
Consider race conditions and performance of the host running the test.

**Comment by billburgess** [comment]
Crazy. Setting the switch manually fails too. Like something is holding up the cell.
```
2018-11-07 14:55:49.526118-0600 simpleinout[12762:26824] Faking turning switch ON
/Users/distiller/project/SimpleInOutTests/LocationListViewControllerTests.m:56: error: -[LocationListViewControllerTests testAddBeacon] : Failed to toggle switch to "ON"; instead, it was "OFF"
<unknown>:0: error: -[LocationListViewControllerTests testAddBeacon] : failed: caught "KIFFailureException", "Failed to toggle switch to "ON"; instead, it was "OFF""
```

Similar to the swipe action, the manual setting of the switch is just telling me it can't (or the view won't). I know the view is rendered since the switch is drawn in code and I wait for another view in the table view to appear before attempting to alter the switch.

**Comment by RafaelPlantard** [comment]
I'm facing some tilt on swipe view too, when swipe is just freezing the screen
